### PR TITLE
Default models

### LINKED ISSUE #18: Syntax Error in raw_contacts Model
Running cht-pipeline in dataemon throws an SQL syntax error:

```sh
13:17:46  Database Error in model raw_contacts (models/contact_views/raw_contacts.sql)
13:17:46    syntax error at or near ";"
13:17:46    LINE 10: ...ct_hospital'::text, 'health_center'::text, 'person'::text]);
13:17:46                                                                           ^
13:17:46    compiled Code at target/run/pipeline/models/contact_views/raw_contacts.sql
13:17:46  
13:17:46  Done. PASS=3 WARN=0 ERROR=1 SKIP=1 TOTAL=5
```

Here's where I think the issue originates from: https://github.com/medic/cht-pipeline/blob/ed63e40a1210363092cbad99150c5e6e2a6d2610/models/contact_views/raw_contacts.sql#L4

### PR DESCRIPTION
**Pull request by njogz**
This extends the default DBT models we provide.

Closes https://github.com/medic/cht-pipeline/issues/14

Closes #18

### CODE REVIEW

**Comment by samuelimoisili** [comment]
@lorerod, if you want to test this branch, you can run 'cht-sync' and update the `.env` file with `CHT_PIPELINE_BRANCH_URL=https://github.com/medic/cht-pipeline.git#14-default-models`.

File: models/reports_tables/reports.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    couchdb.doc ->> '_id'::text AS uuid,
+    couchdb.doc ->> 'form'::text AS form,
+    couchdb.doc -> 'geolocation' AS location,
+    couchdb.doc ->> 'reported_date'::text AS reported_date,
+    couchdb.doc #>> '{contact,_id}'::text[] AS contact_uuid
+FROM {{ ref("couchdb") }}
+WHERE (couchdb.doc ->> 'type'::text) = 'data_record'
+
+{% if is_incremental() %}
+    AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FROM {{ this }}), True)
```
Comment (samuelimoisili) [inline models/reports_tables/reports.sql:17]: Wrong use of aggregate according to DBT.

```sh
15:48:19  Database Error in model reports (models/reports_tables/reports.sql)
15:48:19    aggregate functions are not allowed in WHERE
15:48:19    LINE 25:     AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FR...
```

File: models/reports_tables/reports_by_location.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    couchdb.doc ->> 'form'::text AS form,
+    CAST (coalesce(couchdb.doc #>> '{geolocation,longitude}', '36.826275') AS NUMERIC) AS longitude,
+    CAST (coalesce(couchdb.doc #>> '{geolocation,latitude}', '-1.1792353') AS NUMERIC) AS latitude,
+    couchdb.doc ->> 'reported_date'::text AS reported_date,
+    couchdb.doc #>> '{contact,_id}'::text[] AS contact_uuid
+FROM {{ ref("couchdb") }}
+WHERE (couchdb.doc ->> 'type'::text) = 'data_record'
+
+{% if is_incremental() %}
+    AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FROM {{ this }}), True)
```
Comment (samuelimoisili) [inline models/reports_tables/reports_by_location.sql:18]: Wrong use of aggregate function according to DBT

```sh
15:49:34    aggregate functions are not allowed in WHERE
15:49:34    LINE 26:     AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FR...
```

**Review by samuelimoisili — CHANGES_REQUESTED** [review]

File: models/user_tables/chws.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    doc->>'reported_date' AS reported_date,
+    doc->>'patient_id' AS patient_id,
+    *
+FROM {{ ref('couchdb') }}
+WHERE
+    doc->>'type' = 'person'
+
+{% if is_incremental() %}
+    AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FROM {{ this }}), True)
+{% endif %}
```
Comment (samuelimoisili) [inline models/user_tables/chws.sql:25]: NL

**Review by samuelimoisili — CHANGES_REQUESTED** [review]

File: models/households/household_visits.sql [hunk 4]
```diff
@@ -0,0 +1,8 @@
+SELECT couchdb.doc
+FROM {{ ref("couchdb") }}
+WHERE (couchdb.doc ->> 'type'::text) = ANY
+      (ARRAY ['task'::text, 'form'::text])
+
+{% if is_incremental() %}
+    AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FROM {{ this }}), True)
+{% endif %}
```
Comment (lorerod) [inline models/households/household_visits.sql:8]: NL

**Review by lorerod — APPROVED** [review]

File: models/reports_tables/reports_by_location.sql (hunk 2 above)
Comment (lorerod) [reply models/reports_tables/reports_by_location.sql:18]: I'm getting the same error as @samuelimoisili

**Review by samuelimoisili — APPROVED** [review]

**Pull request merged**